cli/valet.php

```diff
@@ -139,6 +139,25 @@
 })->descriptions('Secure the given domain with a trusted TLS certificate');
 
 /**
+ * Check if a given domain has a trusted TLS certificate.
+ */
+$app->command('secured [domain]', function ($domain = null) {
+    if (!$domain) {
+        warning('Try the following syntax command: valet secured [domain]');
+    }
+
+    $domains = Site::secured();
+
+    foreach ($domains as $item) {
+        $host = explode('.', $item)[0];
+
+        if ($domain && $host === $domain) {
+            info('The ['.$item.'] site has been secured with a TLS certificate.');
+        }
+    }
+})->descriptions('List all secured domain with a trusted TLS certificate');
```

Comment:
I take that back ... 

While looking at your additions of `secured [domain]` and `secures` it strikes me that now there are 4 variants of `secure` in valet commands, and this might not be a good thing.

- secure [domain]
- unsecure [domain]
- secured [domain]
- secures

That feels like it may breed confusion.

Is there a way to just call `isSecured` without needing the `secures`/`secured` commands?